src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -444,6 +491,25 @@ private static boolean hasLineBreakBefore(DetailAST rightCurly) {
     }
 
     /**
+     * Checks if right curly is alone on line in token the lambda.
+     * @param details for validation.
+     * @return true, if right curly is alone on line.
+     */
+    private static boolean isAloneLambda(Details details) {
```

Comment:
@MaksimP , please explain why this method is so specific to LAMBDA ?
if not specific, then name of it should be general, does not contain "Lambda".